c7n/filters/core.py

```diff
@@ -76,6 +76,16 @@ def operator_ni(x, y):
     return x not in y
 
 
+def operator_ni_list_elem(x, y):
+    if isinstance(y, list):
```

Comment:
what if `y` isn't a list?  we should treat it the same as a list with one element.